File: WebApi/api/api/Models/CatalogSize.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace api.Models
{
    public class CATALOG_SIZE
    {
        [Key]
        public long catalog_size_id { get; set; }
        public long catalog_id { get; set; }
        public long catalog_type_id { get; set; }
        public int sort_seq { get; set; }

        [StringLength(10)]
        public string pdsize_code { get; set; }

        [StringLength(15)]
        public string created_by { get; set; }

        public DateTime created_at { get; set; }

        [StringLength(15)]
        public string updated_by { get; set; }

        public DateTime updated_at { get; set; }
    }
}

File: web/src/app/_model/catalog-size.ts
import { AppSetting } from '../_constants/app-setting';

export class CatalogSizeView 
{
    public catalog_size_id: number = undefined;
    public catalog_id: number = undefined;
    public catalog_type_id: number = undefined;
    public pdtype_code: string = "";
    public sort_seq: number = 0;
    public pdsize_code: string = "";
    public pdsize_name: string = "";
    public created_by: string = "";
    public created_at: any = null;
    public updated_by: string = "";
    public updated_at :any= null;  
} 

export class CatalogSizeSearchView
{
    public pageIndex: number = 1;
    public itemPerPage: number = AppSetting.itemPerPage;
    public catalog_id: number = undefined;
    public type_id: number = undefined;
    
}